gradle.properties

```diff
@@ -23,4 +23,4 @@
 
 org.gradle.daemon=true
 org.gradle.parallel=true
-org.gradle.jvmargs=-XX:MaxPermSize=1024m
+org.gradle.jvmargs=-Xmx5120M
```

Comment:
@robertpainsi @p4p4 
Well, it has nothing to do with the object grouping - it's just one of the things you encounter during developing :)
Actually it should speed up development builds, especially with Instant Run - it should be set to a minimum of 2GB that faster builds can be done with DexInProcess. 

There's a good article: 
https://medium.com/google-developers/faster-android-studio-builds-with-dex-in-process-5988ed8aa37e#.jjwhs8idz

It also states that it should correspond to the javaMaxHeapSize option in build.gradle, which currently is 4GB. So maybe we should change it to -Xmx4096M. 
But I can also remove it here and create a separate ticket, since it has nothing to do with object grouping ;) What do you think?